README.md

```diff
@@ -10,6 +10,11 @@
 ```sh
 npx supported <path/to/node_module>
 npx supported <[array/of/node_modules]>
+// Generate token using https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
+npx supported https://github.com/stefanpenner/supported
+npx supported https://github.com/stefanpenner/supported/tree/some-branch
+npx supported https://test.githubprivate.com/stefanpenner/supported -t $TOKEN
```

Comment:
The `GITHUB_TOKEN` environment variable is a convention which we can piggy back on [more details](
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret)

I have fine having `-t` as a way to override, but if a user has `$GITHUB_TOKEN` already set It would be great if we could seamlessly use that.

What do you think?

Reply:
We may also want to provide a link to documentation which helps people generate said token.

Reply:
We have documentation to create documents here and in the error messages as well. https://github.com/stefanpenner/supported/pull/1/files/a4dd4064e13150cba88efa07a1c6eb35d0c7a01b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R13

Reply:
Sorry for the delayed response on this.
I checked to see if I can find `GITHUB_TOKEN` in my local. I couldn't find this. Do you have it in your local environment?

Reply:
Added documentation in the readme file